frappe/utils/boilerplate.py

```diff
@@ -127,12 +127,12 @@ def make_boilerplate(dest, app_name):
 # ------------------
 
 # include js, css files in header of desk.html
-# app_include_css = "/assets/{app_name}/css/{app_name}.css"
-# app_include_js = "/assets/{app_name}/js/{app_name}.js"
+# app_include_css = "/assets/css/{app_name}.css"
```

Comment:
Was working on Schools with @neilLasrado , noticed that it wasn't pulling from that dir.